src/components/notifications/AppNotifications/AppNotificationsCardMobile/index.tsx

```diff
@@ -1,19 +1,18 @@
 import { useContext, useEffect, useMemo, useRef } from 'react'
 
+import { useSubscription } from '@web3inbox/react'
 import { useParams } from 'react-router-dom'
 
 import Button from '@/components/general/Button'
 import CheckMarkIcon from '@/components/general/Icon/CheckMarkIcon'
 import Text from '@/components/general/Text'
 import SettingsContext from '@/contexts/SettingsContext/context'
-import W3iContext from '@/contexts/W3iContext/context'
 
 import './AppNotificationsCardMobile.scss'
 
 const AppNotificationsCardMobile: React.FC = () => {
-  const { topic } = useParams<{ topic: string }>()
-  const { activeSubscriptions, notifyClientProxy } = useContext(W3iContext)
-  const app = activeSubscriptions.find(mock => mock.topic === topic)
+  const { domain } = useParams<{ domain: string }>()
+  const { data: app } = useSubscription(undefined, domain)
```

Comment:
Wouldn't be better to name the exported data as `subscription` instead of `app` since the `data` represents the subscription object?

Reply:
`app` and `subscription` are used interchangeably in the codebase. But  ya I can do that